### PR TITLE
Feature/donate-now-hero-section

### PR DESCRIPTION
**Pull request by misspelz**
image
image

### CODE REVIEW

**Review by edustipend-root — CHANGES_REQUESTED** [review]
Great work @misspelz! Thank you for this
Added a few comments, please make the minor changes so I can get this merged in.

File: src/assets/ade.png
[the remark this answers is not in the document]
Comment (misspelz) [reply src/assets/ade.png:1]: Deleted

File: src/assets/habby.png
[the remark this answers is not in the document]
Comment (misspelz) [reply src/assets/habby.png:1]: Done

File: src/pages/support-a-learner/SupportALearner.module.css
[the remark this answers is not in the document]
Comment (misspelz) [reply src/pages/support-a-learner/SupportALearner.module.css:1]: Deleted

File: src/sections/SupportALearner/HeroSection/HeroSection.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import Header from '../../../components/Header';
+import Text from '../../../components/Text';
+import Button from '../../../components/Button';
+import quoteU from '../../../assets/quoteup.png';
+import quoteD from '../../../assets/quotedown.png';
+import { NavHashLink } from 'react-router-hash-link';
+import Container from '../../../components/Container';
+
+export const HeroSection = () => {
+  const quotesArray = [
```
[the remark this answers is not in the document]
Comment (misspelz) [reply src/sections/SupportALearner/HeroSection/HeroSection.jsx:15]: I have moved it to the constant file

File: src/sections/SupportALearner/HeroSection/contants.js
[the remark this answers is not in the document]
Comment (misspelz) [reply src/sections/SupportALearner/HeroSection/contants.js:1]: I have moved an array of the beneficiaries

File: src/sections/SupportALearner/HeroSection/HeroSection.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      setAnimationKey((prevKey) => prevKey + 1);
+    }, 10000);
+
+    return () => clearInterval(intervalId);
+  }, []);
+
+  const currentQuote = quotesArray[currentQuoteIndex];
+
+  return (
+    <main  className={styles.bg}>
```
[the remark this answers is not in the document]
Comment (misspelz) [reply src/sections/SupportALearner/HeroSection/HeroSection.jsx:51]: Okay, understood
I have used div

File: src/sections/SupportALearner/HeroSection/HeroSection.jsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+import quoteU from '../../../assets/quoteup.png';
+import quoteD from '../../../assets/quotedown.png';
+import { NavHashLink } from 'react-router-hash-link';
+import Container from '../../../components/Container';
+
+export const HeroSection = () => {
+  const quotesArray = [
+    {
+      content:
+        "I didn't have sufficient data to pull through the duration of my course, but Edustipend came to my rescue. And now I made it to the final stage, where my team and I organized clean up program, and also provided character bins to curb plastic waste in our University( the project was achieved through grants from LEAP.",
```
[the remark this answers is not in the document]
Comment (misspelz) [reply src/sections/SupportALearner/HeroSection/HeroSection.jsx:18]: I have pasted the right one

File: src/sections/SupportALearner/HeroSection/HeroSection.jsx [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    },
+    {
+      content:
+        'Edustipend has been a constant support throughout my career journey, allowing me to apply for the stipend multiple times. During my UI/UX designer online program, I was fortunate to be a data beneficiary, enhancing my learning experience. Joining the Edustipend community facilitated connections with like-minded individuals, and monthly study tasks became a valuable part of my routine.',
+      author: 'Sanusi Habeebat',
+      image: Habby
+    },
+    {
+      content:
+        "The laptop stipend was a huge help to me during my final year at the university when I was working on my research project. The Bank of America Investment Banking virtual experience program on Forage provided me with a unique opportunity to gain practical knowledge about finance. Additionally, the ALX AI Career Essentials course I'm currently taking is equipping me with valuable skills to explore the exciting world of AI and be more productive in my career.",
```
[the remark this answers is not in the document]
Comment (misspelz) [reply src/sections/SupportALearner/HeroSection/HeroSection.jsx:30]: Updated

File: src/sections/SupportALearner/HeroSection/HeroSection.jsx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+import quoteD from '../../../assets/quotedown.png';
+import { NavHashLink } from 'react-router-hash-link';
+import Container from '../../../components/Container';
+
+export const HeroSection = () => {
+  const quotesArray = [
+    {
+      content:
+        "I didn't have sufficient data to pull through the duration of my course, but Edustipend came to my rescue. And now I made it to the final stage, where my team and I organized clean up program, and also provided character bins to curb plastic waste in our University( the project was achieved through grants from LEAP.",
+      author: 'Isaac Etor',
```
[the remark this answers is not in the document]
Comment (misspelz) [reply src/sections/SupportALearner/HeroSection/HeroSection.jsx:19]: Yes
I just informed Joshua.
He will get it across to me

File: src/sections/SupportALearner/HeroSection/__tests__/HeroSection.test.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+      expect(screen.getByText(Texts.CONTENT)).toBeInTheDocument();
+
+      expect(screen.getByText(Texts.SUPPORT_TEXT)).toBeInTheDocument();
+
+      expect(screen.getByText(Texts.DONATE_TEXT)).toBeInTheDocument();
+    });
+  });
+
+  /**
```
Comment (edustipend-root) [inline src/sections/SupportALearner/HeroSection/__tests__/HeroSection.test.js:30]: Let's pick this up in a separate PR.

File: src/sections/SupportALearner/HeroSection/contants.js [hunk 7]
```diff
@@ -0,0 +1,55 @@
+import Isaac from '../../../assets/issaac.png';
+import Habby from '../../../assets/habby.png';
+import Ruth from '../../../assets/ruthty.png';
+import Idrees from '../../../assets/Idrees.png';
+import Maria from '../../../assets/Ochi.png';
+
+export const quotesArray = [
+  {
+    content:
+      'Edustispend supported my career advancement by assisting with a portion of my course fees, thereby providing vital financial support and facilitating my pursuit of educational objectives. Their support has been valuable to me and to many others.',
```
Comment (edustipend-root) [inline src/sections/SupportALearner/HeroSection/contants.js:10]: There's a typo here: `Edustipend` not `Edustispend`

**Review by edustipend-root — APPROVED** [review]
@misspelz, I'm approve this for now so we can get it merged in but we need to make some changes in a follow-up PR:

- We need to update the quote component on the image such that when user hovers over the text, it shows the full text because right now the text is blocking the image especially on mobile. Let me know if this makes sense to you.
- 
- The typo change

File: src/sections/SupportALearner/HeroSection/contants.js (hunk 7 above)
Comment (2gedareactfrontend) [reply src/sections/SupportALearner/HeroSection/contants.js:10]: Corrected

File: src/sections/SupportALearner/HeroSection/__tests__/HeroSection.test.js (hunk 6 above)
Comment (2gedareactfrontend) [reply src/sections/SupportALearner/HeroSection/__tests__/HeroSection.test.js:30]: Okay

**Comment by 2gedareactfrontend** [comment]
> @misspelz, I'm approve this for now so we can get it merged in but we need to make some changes in a follow-up PR:
> 
> * We need to update the quote component on the image such that when user hovers over the text, it shows the full text because right now the text is blocking the image especially on mobile. Let me know if this makes sense to you.
> * The typo change

I understand but what about the mobile view? Will users be able to hover ?

**Pull request merged**